app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ErrorsController < ApplicationController
+  # skip_before_action :authenticate_request
+
+  def routing_error
+    head :not_found
```

Comment:
We did this because in production this redirection is done by default, but in stage it raises an exception. We will change the environment to production and remove this controller and route.